bot.py

```diff
@@ -106,10 +108,26 @@ async def pomodoro(ctx):
     # Notify the user after 5 minutes that Pomodoro has ended
     await ctx.send(ctx.message.author.mention + " Pomodoro has ended!")
 
-def getQuote(tags=["education", "success"]):  # default arguments
-    # Fetch quotes using an API
-    # https://github.com/lukePeavey/quotable
-    pass
+def getQuote(tags=["inspirational", "success"]):  # default arguments
+    # Get json response containing quote data from the api
+    response = urllib.request.urlopen(f"https://api.quotable.io/random?tags={tags[0]}|{tags[1]}").read()
```

Comment:
["inspirational", "success" ] are just default arguments. Thus, you cannot use tags[0] and tags[1], because the user may input any number of tags (as a list). Also, validate them from the list of tags that are available, and return an appropriate response if the user passes an invalid tag.

Reply:
ok i will try to implement that